integration-test/config.js

```diff
@@ -0,0 +1,10 @@
+export default {
+    spec_dir: 'integration-test',
```

Comment:
Is there a need for this to be JS, curious mainly.

Reply:
I'd ask the opposite - why keep it as JSON which is incredibly restrictive if the `.js` version is valid for Jasmine - opens the door for logic/comments/eslint etc.

Basically I don't have a great reason, other that, "why not"?

Reply:
> why keep it as JSON which is incredibly restrictive

This is the feature, don't open yourself up to flexibility you don't need yet.

Either way it's fine, I'd be pushing back further for .eslintrc or similar files.